components/services/bat_ledger/public/interfaces/bat_ledger.mojom

```diff
@@ -146,6 +144,7 @@ interface BatLedgerClient {
 
   [Sync]
   SetTimer(uint64 time_offset) => (uint32 timer_id);
+  KillTimer(uint32 timer_id);
```

Comment:
Probably just my lack of understanding, but elsewhere in the code a comment indicated that `KillTimer` was synchronous. I was expecting to see it marked with `[Sync]` here.